setup.py

```diff
@@ -67,7 +67,7 @@ def _read_file(filename):
       'Record',
       'RestrictedPython',
       'ZConfig >= 2.9.2',
-      'ZODB3',
+      'ZODB',
```

Comment:
We should be adding a pin on `BTrees`, since this package imports it explicitly.